backend/integrations/discord/cogs.py

```
@@ -391,3 +399,124 @@ async def list_indexed_repos(self, interaction: discord.Interaction):
+                discord_username=getattr(user, "name", str(user)),
+                avatar_url=str(user.avatar.url) if getattr(user, "avatar", None) else None,
+            )
+
+            # Already verified: send final hand-off and finish
+            if getattr(profile, "is_verified", False) and getattr(profile, "github_id", None):
+                try:
+                    await user.send(build_verified_welcome(profile.github_username))
+                    await user.send(build_verified_capabilities_intro(profile.github_username))
+                    await user.send(embed=build_final_handoff_embed())
+                except Exception:
+                    pass
```

Ruff (0.14.1)

`try`-`except`-`pass` detected, consider logging the exception

(S110)